src/location.c

```diff
@@ -718,10 +739,13 @@ location_class_init (LocationClass *klass)
 
 GDBusInterfaceSkeleton *
 location_create (GDBusConnection *connection,
-                 const char *dbus_name)
+                 const char *dbus_name,
+                 gpointer lockdown_proxy)
 {
   g_autoptr(GError) error = NULL;
 
+  lockdown = g_object_ref (lockdown_proxy);
```

Comment:
Odd to take refs and not release them.

Reply:
That did occur to me too. We're leaking stuff all over the place here, anyway. Let me fix this up